lib/datahub-client/data_platform_catalogue/search_types.py

```diff
@@ -11,11 +11,37 @@
 class ResultType(Enum):
     """Result type."""
 
-    TABLE = auto()
-    GLOSSARY_TERM = auto()
-    CHART = auto()
-    DATABASE = auto()
-    DASHBOARD = auto()
+    TABLE = ("Table", "DATASET", "table")
```

Comment:
It might be worth extracting all this result type/subtype stuff to entities.py or a separate module since it seems relevant more broadly (it's not just a search thing)

Reply:
done